tests/integration/test_models_vision_text_config_registry.py

```
@@ -0,0 +1,1679 @@
+# ---------------------------------------------------------------------------
+
+
+def _ok_response(text: str = "Generated text") -> dict:
+    return {"response": text, "done_reason": "stop", "total_duration": 1_000_000_000}
+
+
+def _exhausted_response(text: str = "") -> dict:
+    return {"response": text, "done_reason": "length", "total_duration": 1_000_000_000}
+
+
+def _make_text_model() -> Any:
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_make_text_model`

(ANN401)